### ISSUE TITLE
Addition of complex number converter

### ISSUE THREAD

**Issue by electricman94** [ZoomerAnalytics/xlwings#945]
OS (e.g. Windows 10 or macOS Sierra)
Windows 10

Versions of xlwings, Excel and Python (e.g. 0.11.8, Office 365, Python 3.7)
xlwings=0.11.8
Office 365
Python=3.6.5

Describe your issue (incl. Traceback!)
Unexpected Python   Error: TypeError: Objects of type 'complex' can not be converted to a COM   VARIANT

Include a minimal code sample to reproduce the issue (and attach a sample workbook if required!)
```
import xlwings as xw
@xw.func
def comp():
    return complex(1)
```

**Comment by fzumstein** [comment]
Apparently, complex numbers in Excel are just strings, so you may be able to work around with something like `str(complex(1))[1:-1]` for now.

**Comment by DougAJ4** [comment]
Yes, Excel complex numbers are strings, with little or no support for doing any maths with them.

It would be good to have a VBA complex number type that would return doubles for the real and complex part and that would work with Python complex number operations.
